uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/helpers/WorkPatternHelpersTrait.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+trait CRM_HRLeaveAndAbsences_WorkPatternHelpersTrait {
+
+  public function getWorkDayTypeOptionsArray() {
+    $result = $result = civicrm_api3('OptionValue', 'get', array(
```

Comment:
This is what happens when you just copy and paste the code from somewhere else :P